sdk/textanalytics/ai-text-analytics/src/textAnalyticsResult.ts

```diff
@@ -1,13 +1,70 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { TextDocumentStatistics, TextAnalyticsError } from "./generated/models";
+import {
+  TextDocumentStatistics,
+  TextAnalyticsError as GeneratedTextAnalyticsErrorModel,
+  InnerError
+} from "./generated/models";
 
 /**
  * The result of a text analytics operation on a single input document.
  */
 export type TextAnalyticsResult = TextAnalyticsSuccessResult | TextAnalyticsErrorResult;
 
+/**
+ * An Error Code returned from the Text Analytics service. Possible
+ * values include:
+ *
+ * - 'InvalidRequest'
```

Comment:
this comment feels weird since the type is a string literal union...

Reply:
My thought was that it could be useful for JS but then I realized that just because a user is using JS doesn't mean they don't see the full typed intellisense or our beautiful ref docs.

Once the docs.ms docs are rendered we can check this again because I'm not 100% sure it will show correctly there.